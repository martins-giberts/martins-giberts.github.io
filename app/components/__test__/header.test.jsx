/* globals describe, it, expect */

import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Component from '../header/index'

describe('components/Header test', () => {
  it('should render without throwing an error', () => {
    const wrapper = shallow(<Component/>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
