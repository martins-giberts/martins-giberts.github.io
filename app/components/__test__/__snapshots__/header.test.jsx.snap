// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header test should render without throwing an error 1`] = `
<div
  className="header"
>
  <div
    className="cell"
  >
    <Title
      Kind="h1"
    >
      Martins Giberts
    </Title>
  </div>
  <div
    className="cell"
  >
    <Title
      Kind="h2"
    >
      Web applications developer
    </Title>
  </div>
</div>
`;
